cfme/test_framework/appliance.py

```diff
@@ -6,11 +6,11 @@
 from cfme.utils import conf
 from cfme.utils.appliance import (
     load_appliances_from_config, stack,
-    DummyAppliance, IPAppliance,
+    DummyAppliance,
     ApplianceSummoningWarning)
 from cfme.utils.path import log_path
 from fixtures import terminalreporter
-
+PLUGIN_KEY = "appliance-holder"
```

Comment:
**QUESTION:** Why is that a good name?   I think I just don't know some context here, but to me that's a really mysterious name.   Could you add a comment explaining what a plugin key is?